website/pages/docs/connect/terminating_gateway.mdx

```diff
@@ -0,0 +1,122 @@
+---
+layout: docs
+page_title: Connect - Terminating Gateways
+sidebar_title: Terminating Gateways
+description: >-
+  A terminating gateway enables traffic from services in the Consul
+  service mesh to services outside the mesh. This section details
+  how to configure and run a terminating gateway.
+---
+
+# Terminating Gateways
+
+-> **1.8.0+:**  This feature is available in Consul versions 1.8.0 and newer.
+
+Terminating gateways enable connections from services in the Consul service mesh to
+services outside the mesh. These gateways effectively act as Connect proxies that can
+represent more than one service. They terminate Connect mTLS connections, enforce intentions,
+and forward requests to the appropriate destination.
+
+## Security Considerations
+
+We recommend that terminating gateways not be exposed to the WAN or open internet. This is because terminating gateways
```

Comment:
You might consider moving this to the intro and putting it in a call out box so users don't miss it.

Reply:
Moved it into a call out box but left it at the top of this section. Let me know what you think.